openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx

```diff
@@ -490,14 +484,17 @@ const ConfigureIngestion = ({
         <p className="tw-text-grey-muted tw-mt-1 tw-mb-2 tw-text-sm">
           {t('message.database-service-name-message')}
         </p>
-        <input
-          className="tw-form-inputs tw-form-inputs-padding"
+        <Select
+          allowClear
           data-testid="name"
           id="name"
-          name="name"
-          type="text"
+          mode="tags"
```

Comment:
Can we also add some instruction to let them know that they can add multiple

Reply:
updated description message